### ISSUE TITLE
[sub-issue] CRw: Emergency-Exclamation Mark is ignored by the screenreader

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#320]
![image](https://cloud.githubusercontent.com/assets/7510842/5278114/0ff966d6-7ad6-11e4-94db-a43b4e48e0f4.png)

Emergency-Exclamation Mark is not read by screen readers.

==> Use ARIA "labelledby" or a hidden text - text should be "Alert"
If you use ARIA check it out with the screen reader NVDA http://www.nvaccess.org/download/

**Comment by tonifisler** [comment]
Main issue is #340

**Comment by tonifisler** [comment]
This is a pseudo-class element. It can't be read by the screen reader at all. The attribute `role="alert"` around the whole alert should be sufficient in this case.
